packages/convai-widget-core/tailwind.config.js

```diff
@@ -1,3 +1,57 @@
+const colors = ["base", "accent"];
+const shades = [
+  null,
+  "hover",
+  "active",
+  "border",
+  "subtle",
+  "primary",
+  "error",
+];
+const spacing = [
+  0, 0.5, 1, 1.5, 2, 2.5, 3, 3.5, 4, 5, 6, 7, 8, 9, 10, 11, 12, 14, 16, 20, 24,
+  28, 32, 36, 40, 44, 48, 52, 56, 60, 64, 72, 80, 96,
+];
+const radius = [1, 2, 2.5, 3, 4, 5, 6, 7, 7.5];
+
```

Comment:
I've hit some issues in the past where global styles of the parent page would be set to some non-standard value, and it would be carried out to shadow too, unless reset within. 
I don't remember the detail, but we could test and possibly introduce our won reset.

Reply:
Ah yeah rem must be the thing I remember. Seems like you got that covered!

Note - I would not expect any issues on docs or xi - those use pretty much the same setup with tailwind, etc.